Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -52,6 +52,8 @@ def buildConverters(tableSpec, tableNamespace):
 			converterClass = FeatureParams
 		elif name in ("CIDGlyphMapping", "GlyphCIDMapping"):
 			converterClass = StructWithLength
+		elif name == "Flags":
+			converterClass = STATFlags
```

Comment:
remove this `elif` block, add a `STATFlags: STATFlags` to the `converterMapping` below, then change the entry for the STAT Flags in `otData.py` module so that the type of the converter is `STATFlags` instead of `uint16`

Reply:
Sounds good.